docker.Makefile

```diff
@@ -8,15 +8,15 @@ $(warning WARNING: No docker user found using results from whoami)
 DOCKER_ORG                = $(shell whoami)
 endif
 
-CUDA_VERSION              = 11.3.1
+CUDA_VERSION              = 11.6.1
 CUDNN_VERSION             = 8
 BASE_RUNTIME              = ubuntu:18.04
 BASE_DEVEL                = nvidia/cuda:$(CUDA_VERSION)-cudnn$(CUDNN_VERSION)-devel-ubuntu18.04
 
 # The conda channel to use to install cudatoolkit
 CUDA_CHANNEL              = nvidia
 # The conda channel to use to install pytorch / torchvision
-INSTALL_CHANNEL          ?= pytorch
+INSTALL_CHANNEL          ?= pytorch-nightly
```

Comment:
+1 on @seemethere's comment: please don't change the default (unless there is a good reason)
```suggestion
INSTALL_CHANNEL          ?= pytorch
```

Reply:
Its breaking if I use pytoch  channel, since pytoch-cuda package in not yet uploaded there. Let me upload this package there and use pytorch channel

Reply:
Done Uploaded pytorch-cuda 11.6 to pytorch